docs/developing.md

```diff
@@ -22,11 +22,14 @@ Most of Jupytext's code is written in Python. To develop the Python part of Jupy
 ```
 conda env create --file environment.yml  # or conda env update --file ...
 conda activate jupytext-dev
-python -m ipykernel install --name jupytext-dev --user
-pip install -e .
 ```
 
-We use the [pre-commit](https://pre-commit.com) package to run pre-commit scripts like `black` and `flake8` on the code.
+Install the `jupytext` package in development mode with
+```
+pip install '.[dev]'
```

Comment:
I guess you are missing a `editable` flag here? `pip install -e '.[dev]'`?

Reply:
Oh yes you're right! Thanks